ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-ref.yaml

```diff
@@ -10,8 +10,5 @@ ref:
     requests:
       cpu: 100m
       memory: 200Mi
-  dependencies:
-  - name: "release:target"
-    env: "RELEASE_IMAGE_TARGET"
```

Comment:
@wangke19 I did not see any place reference this ENV, so I remove it from this step.

Reply:
It is used in step https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-ref.yaml#L13-L15

Reply:
But I cannot find where is it used in shell script. it might be useless. I need to raised one pr to remove reference from step.

Reply:
> It is used in step https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-ref.yaml#L13-L15

I know that, I am removing the ENV from the step, or else, I can not migrate it to mult-arch ci from amd ci.